lib/db/backend/leveldb_open.go

```diff
@@ -0,0 +1,226 @@
+// Copyright (C) 2019 The Syncthing Authors.
```

Comment:
Local git considers leveldb_open.go as a copy of lowlevel.go (52%), thus maybe leave the copyright year at 2018.